plugins/base/src/test/kotlin/basic/DRITest.kt

```diff
@@ -10,7 +10,7 @@ import org.junit.jupiter.api.Test
 
 class DRITest : AbstractCoreTest() {
     @Test
-    fun `#634`() {
+    fun issue634() {
```

Comment:
what was wrong with previous naming convention?

Reply:
it was failing to build on windows